source/stylesheets/base/_variables.scss

```diff
@@ -53,6 +53,18 @@ $base-timing: ease;
 
 // Grid Sizes
 $max-width: 1000px;
+$neat-custom-grid: (
```

Comment:
I'd just rearrange these so they go from small to large. It's easier to read that way. I'd also include a line break between each of them.